features/data-urls.yml

```diff
@@ -1,6 +1,7 @@
 name: Data URLs
 description: Data URLs are URLs that start with the `data:` scheme and which directly contain data rather than point to remote resources.
 spec: https://www.rfc-editor.org/rfc/rfc2397
+caniuse: datauri
 status:
   compute_from: http.data-url
```

Comment:
To align statuses, we'd have to include `http.data-url.html_files` in the status computation. That would shift the Baseline date forward (2015→2020). Probably good?